bin/po2swift.js

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env node
+
+const gettextParser = require( 'gettext-parser' ),
+	fs = require( 'fs' );
+
+function po2Swift( poInput ) {
+	const po = gettextParser.po.parse( poInput );
+	const translations = po.translations[ '' ];
+	const swiftStrings = Object.values( translations ).map( ( translation, id ) => {
+		if ( translation.msgid === '' ) {
+			return null;
+		}
+		const encodedValue = JSON.stringify( translation.msgid );
+		const encodedComment = JSON.stringify( translation.comments.extracted || '' );
+		let localizedStringInSwift = `let string${ id } = NSLocalizedString(${ encodedValue }, comment: ${ encodedComment })\n`;
+		if ( translation.msgid_plural ) {
+			const encodedValuePlural = JSON.stringify( translation.msgid_plural );
+			localizedStringInSwift += `let string${ id }Plural = NSLocalizedString(${ encodedValuePlural }, comment: ${ encodedComment })\n`;
+		}
+		return localizedStringInSwift;
+	} ).filter( Boolean );
+	return swiftStrings.join( '' );
+}
+
+if ( require.main === module ) {
+	if ( process.stdin.isTTY ) {
+		const potFileName = process.argv[ 2 ];
+		const destination = process.argv[ 3 ];
+		const potFileContent = fs.readFileSync( potFileName );
+		const swiftOutput = po2Swift( potFileContent, process.argv[ 3 ] );
+		fs.writeFileSync( destination, swiftOutput );
+	} else {
```

Comment:
Can you explain what the non-TTY variant is for? I'm not that familiar with node on the cli, and I don't understand why there are two variants

Reply:
This is so the script can work with pipes: `cat gutenberg.pot | node bin/po2swift.js`.
I started with this approach because it was easier to test, and I wasn't sure which one I was going to use in package.json. It's not needed anymore so we could remove it but I figured it would be a nice addition